pkgs/applications/editors/standardnotes/standardNotes.nix

```diff
@@ -0,0 +1,31 @@
+with import <nixpkgs> {};
+
+stdenv.mkDerivation rec {
+  version = "2.3.11";
+  name = "standardNotes-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/standardnotes/desktop/releases/download/v${version}/standard-notes-${version}-x86_64.AppImage";
+    sha256 = "1zbkmxsipmprrbgjmszg33w7cqs5v5zrcwknkqwybj9q41addsql";
+  };
+
+  buildInputs = [ appimage-run  ];
+
+  phases = " installPhase";
+
+   installPhase = ''
+    mkdir -p $out/bin
+    cp $src $out/standardNotes.AppImage
+    echo "#!usr/bin/env bash" > $out/bin/standardnotes
```

Comment:
use `#!/bin/sh` instead of bash. It does not depend on the PATH and is guaranteed to exist by posix.

Reply:
Or alternatively, if you are doing anything bash-specific, use "${bash}/bin/bash".

Reply:
Thank you for the info!